laborator/content/apel-functii/solution/4-print-string-len/print-string-len.asm

```diff
@@ -0,0 +1,31 @@
+section .data
+    mystring db "This is my string", 0
+    print_format db "String length is %d", 10, 0
+
+section .text
+
+extern printf
+global main
+
+main:
+    push ebp
+    mov ebp, esp
+
+    mov eax, mystring
+    xor ecx, ecx
+test_one_byte:
+    mov bl, byte [eax]
+    test bl, bl
+    jz out
+    inc eax
+    inc ecx
+    jmp test_one_byte
+
+out:
+    push ecx
```

Comment:
Instead of simply calling `printf()` directly, it may be useful for students to see the differences between `PRINTF32` and the manual function call. So keep the same print as in the skeleton:

```asm
    PRINTF32 `[PRINTF32]: %d\n[printf]: \x0`, ecx
```